sdk/src/CrashReporter.ts

```diff
@@ -200,6 +208,48 @@ export default class CrashReporter {
       return;
     }
 
+    if (isFatal) {
+      setCurrentTags(getCurrentTags().concat('UnhandledError'))
+    }
+
+    const stack = await this.cleanStackTrace(error);
+
+    const payload = await this.generateCrashReportPayload(error, stack);
+
+    this.managePayload(payload);
+  }
+
+  /**
+   * Processes a manually sent error (using local tags, not global).
```

Comment:
Manually reported errors still include the global tags. 
ManualSend -> LocalTags + GlobalTags -> Post
AutomaticSend -> GlobalTags -> Post